docs/source/contents/4_draft_mapping.rst

```diff
@@ -1399,6 +1398,164 @@ https://digital.lib.utk.edu/collections/islandora/object/utsmc:725/datastream/MO
+Islandora Metadata Interest Group has come up with the solution of using the dcterms resource type of "Collection." In
+this situation we will need multiple triples to preserve the information currently present - one for indicating the record is
+for a collection and one (or more) for indicating prevalent resource type(s) in the collection. In MODS typeOfResource is
+a repeatable field. Note that we will need to make sure that we do not repeat the collection resource type in cases
+where there are multiple typeOfResource[@collection="yes"] instances.
+
++----------------------------+---------------+--------------------------------------------------+--------------------+
+| collection="yes"           | dcterms:type  | <http://id.loc.gov/vocabulary/resourceTypes/col> | Collection         |
++----------------------------+---------------+--------------------------------------------------+--------------------+
+
+Justification
+^^^^^^^^^^^^^
+
+We need to be able to distinguish between an item and collection resource, so retaining this information is necessary.
+
+Xpath
+^^^^^
+
+:code:`mods:typeOfResource[@collection="yes"]`
+
+Decision
+^^^^^^^^
+
+Here's a complex example that includes two <typeOfResource> values - `gsmrc:smhc <https://digital.lib.utk.edu/collections/islandora/object/gsmrc%3Asmhc/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <typeOfResource collection="yes">text</typeOfResource>
+    <typeOfResource collection="yes">still image</typeOfResource>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:type <http://id.loc.gov/vocabulary/resourceTypes/col> ;
+        dcterms:type <http://id.loc.gov/vocabulary/resourceTypes/txt> ;
+        dcterms:type <http://id.loc.gov/vocabulary/resourceTypes/img> .
+
+Missing typeOfResource value
+----------------------------
+
+Use case
+^^^^^^^^
+
+Currently 9,993 records are missing a typeOfResource value. The affected collections include Volunteer Voices (not entire
+collection), Roth, the Howard Baker Speeches and Remarks, Great Smoky Mountains Colloquy, and the Great Smoky Mountains Postcard Collection. We can consider if we would like to apply a blanket value to a collection at the time
+of migration. For monolithic collections like Roth and Baker, this would be easy to achieve (roth = "still image" and
+baker = "text" in MODS). For collections with varied formats, like Volunteer Voices, this will not be possible.
+
+Justification
+^^^^^^^^^^^^^
+
+Given that the Digital Collections home page currently uses typeOfResource to initially limit searches, it would be
+beneficial for this value to be more consistently present. It would also assist with discovery in DPLA.
+
+Xpath
+^^^^^
+
+:code:`not(mods:typeOfResource)`
+
+Decision
+^^^^^^^^
+
+During migration we will plan to add typeOfResource on a collection basis if possible. See the chart below for decisions.
```

Comment:
I'd agree. I think I was trying to be less wishy-washy, but the fact of the matter is we don't know exactly when this work will be done and it will likely be post migration. I'll commit a change.

Reply:
The sentence following this does clarify that post migration is possible, but yes, the way it is written it makes it seem that during migration is likely.